app/javascript/components/Dashboard/Tooltips.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Tooltip = ({ feature }) => {
+  const { id } = feature.properties;
+  console.log("tool", feature)
+  return (
+    <div id={`tooltip-${id}`}>
+      <strong>Source Layer:</strong> {feature.layer["source-layer"]}
+      <br />
+      <strong>Layer ID:</strong> {feature.layer.id}
+    </div>
+  );
+};
+
+export default Tooltip;
```

Comment:
```suggestion
export { Tooltip };
```
Why not like this? We have been using this method of exporting everywhere